reference/supported-integrations.md

```diff
@@ -24,7 +24,7 @@ Or, you can refer to the [How to Write A Driver](../tutorial/write-a-driver) gui
 
 ## Messaging
 * Email (MS Graph, Gmail, SMTP, AWS SES)
-* SMS (MessageMedia)
+* SMS (MessageMedia, Whispir, Amazon SMS)
```

Comment:
Where did the Amazon reference come from? I assume this is potentially reference Amazon _SNS_ (which can be used to send SMS's), but cannot see an active integration for this.

Reply:
Correcto, its been done for NSWH just needs to be pulled out of private, but it exists so happy for it to be on the list.